modules/header/react/spec/__snapshots__/Header.snapshot.tsx.snap

```diff
@@ -232,41 +193,24 @@ exports[`Dub Header Snapshots renders a default header 1`] = `
 }
 
 <div
-  className="emotion-10 emotion-11"
+  className="emotion-4 emotion-5"
 >
   <div
-    className="emotion-6 emotion-7"
+    className="emotion-0 emotion-1"
   >
     <span>
-      <div
-        className="emotion-4 emotion-5"
-      >
-        <div
-          className="emotion-2 emotion-3"
-          title=""
-        >
-          <div
-            className="emotion-0 emotion-1"
-            dangerouslySetInnerHTML={
-              Object {
-                "__html": "<img src=\\"//design.workday.com/images/ck-dub-logo-blue.svg\\">",
-              }
-            }
-            title=""
-          />
-          
-        </div>
-      </div>
+      MockDubLogoTitle
+	themeColor: 0
```

Comment:
@vibdev @anicholls 
mocked some components from other modules so that we can see the important details (name and props) but not have the snapshot blown up with all the internals of that component (css and structure). i can revert if you guys prefer but personally find that it makes the snapshot for this component more readable